doc/context-command.md

```diff
@@ -55,6 +55,12 @@ Command output can be partially tested, this is useful is you don't care about o
   """
 ```
 
+Or you can only search for a string
```

Comment:
`Or you can test the complete output:`
It's not `searching for a string` as you check the all string output